stable/k8s-secret-updater/README.md

```diff
@@ -50,3 +50,11 @@ See the [k8s-secret-updater](https://github.com/fairfaxmedia/k8s-secret-updater)
 | `ingress.containerPort` | `80`                         | ..          |
 | `ingress.host`          | `secret-updater.example.com` | ..          |
 | `ingress.annotations`   | `{}`                         | ..          |
+
+### Service Account
+
+| Parameter               | Default | Description |
+| ----------------------- | ------- | ----------- |
+| `serviceAccount.create` | `false` | Create a service account with `.Chart.Name` as the name or if `serviceAccount.name`  set will use this as the name |
```

Comment:
Common toggle is often`enabled` which can be `true` or `false`.

Reply:
This was already created in Chart version 2.1.0. Might be best to create a seperate PR for this change